contrib/opencensus-ext-azure/CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
 # Changelog
 
 ## Unreleased
+- Supported server performance breakdown by operation name
```

Comment:
@lzchen I just noticed the changelog is not reflecting your metrics work, please have this fixed, thanks.

Reply:
Are you referring to the PR number?